plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestingKuduServer.java

```diff
@@ -71,9 +101,23 @@ public void close()
         try (Closer closer = Closer.create()) {
             closer.register(master::stop);
             tServers.forEach(tabletServer -> closer.register(tabletServer::stop));
+            closer.register(toxiProxy::stop);
+            closer.register(network::close);
         }
         catch (IOException e) {
             throw new RuntimeException(e);
         }
     }
+
+    private static String getHostIPAddress()
+    {
+        try {
+            // This is roughly equivalent to setting the KUDU_QUICKSTART_IP defined here: https://kudu.apache.org/docs/quickstart.html#_set_kudu_quickstart_ip
+            // What if there are multiple network interfaces with multiple IPV4 addresses bound? Choose one.
```

Comment:
I don't believe it matters for localhost.

Reply:
However if needed we can:

```java
Enumeration<NetworkInterface> networkInterfaceEnumeration = NetworkInterface.getNetworkInterfaces();
while (networkInterfaceEnumeration.hasMoreElements()) {
    for (InterfaceAddress interfaceAddress : networkInterfaceEnumeration.nextElement().getInterfaceAddresses()) {
        if (interfaceAddress.getAddress().isSiteLocalAddress()) {
            return interfaceAddress.getAddress().getHostAddress();
        }
    }
}
```

Reply:
Reminder to either remove this comment (if loop-back is just what we need) or address this.

Reply:
Took your suggestion and added a comment explaining why

Reply:
Thanks for explaining why loop-back won't work. I didn't even realise at first. 😄